frontend/app/components/team-show.js

```diff
@@ -40,7 +40,18 @@ export default class TeamShowComponent extends Component {
       headers: {
         "X-CSRF-Token": $('meta[name="csrf-token"]').attr("content")
       }
-    }).then(() => (this.args.team.favourised = !this.args.team.favourised));
+    }).then(() => {
+      this.args.team.favourised = !this.args.team.favourised;
+      if (this.navService.isShowingFavourites) {
+        let teams = this.navService.availableTeams.toArray();
+        if (this.args.team.favourised) {
+          teams.pushObject(this.args.team);
+        } else {
+          teams.removeObject(this.args.team);
+        }
+        this.navService.availableTeams = teams;
```

Comment:
We could possibly save some lines here 😉

Reply:
Considering what I wrote above... can we? :sweat_smile: